Makefile.in

```diff
@@ -74,29 +74,29 @@ static:  $(top_builddir)/$(STATIC_LIBRARY)
 
 $(top_builddir)/$(app_name): $(top_builddir)/lsauxv.o $(top_builddir)/$(SHARED_LINKERNAME_LIB)
 ifeq ($(sysheaders), )
-	@CC@ @CFLAGS@ -Wall -o $(top_builddir)/$(app_name) $(top_builddir)/lsauxv.o -L$(top_builddir) -lauxv -lpthread
+	@CC@ @CFLAGS@ @LDFLAGS@ -Wall -o $(top_builddir)/$(app_name) $(top_builddir)/lsauxv.o -L$(top_builddir) -lauxv -lpthread
 	@echo
 else
-	@CC@ @CFLAGS@ -Wall -o $(top_builddir)/$(app_name) $(top_builddir)/lsauxv.o -L$(top_builddir) -lauxv -I$(sysheaders) -lpthread
+	@CC@ @CFLAGS@ @LDFLAGS@ -Wall -o $(top_builddir)/$(app_name) $(top_builddir)/lsauxv.o -L$(top_builddir) -lauxv -I$(sysheaders) -lpthread
 	@echo
 endif
 
-#@CC@ @CFLAGS@ -Wall -o $(top_builddir)/$(app_name) $(top_builddir)/lsauxv.o $(top_builddir)/$(SHARED_SONAME_LIB) -lpthread
+#@CC@ @CFLAGS@ @LDFLAGS@ -Wall -o $(top_builddir)/$(app_name) $(top_builddir)/lsauxv.o $(top_builddir)/$(SHARED_SONAME_LIB) -lpthread
 $(top_builddir)/lsauxv.o: $(top_srcdir)/lsauxv.c $(install_headers)/auxv.h
 ifeq ($(sysheaders), )
-	@CC@ @CFLAGS@ -Wall -c -o $(top_builddir)/lsauxv.o $(top_srcdir)/lsauxv.c -I$(install_headers)
+	@CC@ @CFLAGS@ @LDFLAGS@ -Wall -c -o $(top_builddir)/lsauxv.o $(top_srcdir)/lsauxv.c -I$(install_headers)
```

Comment:
IMO, it doesn't make sense to add LDFLAGS to a compile command, e.g. "gcc -c".
Why is it necessary here?

I noticed the same behavior in the next 3 changes too.